common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java

```diff
@@ -274,26 +319,32 @@ private void dropEphemeralNamespaceIfNecessary(ExtensionContext context) throws
         }
     }
 
-    private void runPublicStaticVoidMethods(Class<? extends Annotation> annotation, ExtensionContext context) throws Exception {
-        for (Method method : context.getRequiredTestClass().getMethods()) {
-            if (method.getAnnotation(annotation) != null) {
-                if (!isPublicStaticVoid(method)) {
-                    throw new OpenShiftTestException("@" + annotation.getSimpleName()
-                            + " method " + method.getDeclaringClass().getSimpleName() + "." + method.getName()
-                            + " must be public static void");
+    private void runPublicStaticVoidMethods(Class<? extends Annotation> annotation, ExtensionContext context) {
```

Comment:
The change here will just lead to harder-to-understand exception messages. Since you no longer run this in parallel with additional resources, can you revert this method back to how it looked?